addons/mail/static/src/components/chatter/chatter.xml

```diff
@@ -44,6 +44,7 @@
                             hasScrollAdjust="chatter.hasMessageListScrollAdjust"
                             order="'desc'"
                             threadViewLocalId="chatter.threadView.localId"
+                            scrollableElement="scrollableElement"
```

Comment:
Should use another name, this looks like it is passing the element itself when it is passing a function.